suite-common/graph/src/hooks.ts

```diff
@@ -154,8 +156,10 @@ export function useGraphForAccounts(params: useGraphForAccountsParams): {
     const refetch = useCallback(() => fetchGraphValues({ forceRefetch: true }), [fetchGraphValues]);
 
     useEffect(() => {
-        fetchGraphValues();
-    }, [fetchGraphValues]);
+        if (isDeviceAuthorized) {
+            fetchGraphValues();
+        }
```

Comment:
This needs to work for portfolio tracker also.

Reply:
portfolio tracker device is always authorized (has state)

Reply:
Aw yes, my bad.